docs_dev/releasing.md

```diff
@@ -265,20 +291,9 @@ getting the correct installation instructions etc. Here are the steps to follow:
   not yet exist. We'll have the page for version 1.0 onwards. See 
   [PR #1308](https://github.com/kubeflow/website/pull/1308).)
 
-<a id="create-website-branch"></a>
-### Creating a website branch for the latest major or minor release
-
-We usually create the website branch for the current version close to the date
-of the next upcoming major/minor release. The website branch is in effect an
-archived snapshot of the docs. (We need to wait at least a few weeks after the 
-software release of the current version before creating the website branch, 
-because it takes a while to finish updating the docs.)
 
-If the documentation for a version needs to be fixed after we've created
-the website branch, the changes should be committed to master and then 
-cherry-picked to the proper version branch.
-
-Follow these steps to create a version branch of the website:
+<a id="create-website-branch"></a> 
```

Comment:
While you're here :) please would you remove this sentence from line 289:
" (This is a placeholder for the task, as the page does not yet exist. We'll have the page for version 1.0 onwards. See PR #1308.)"

and replace it with this link:
`https://github.com/kubeflow/website/edit/master/content/docs/reference/version-policy.md`

Reply:
Done. And added instructions on how to get the versions.